pkg/storage/stores/tsdb/single_file_index.go

```diff
@@ -384,7 +385,17 @@ func (i *TSDBIndex) SeriesVolume(ctx context.Context, _ string, from, through mo
 					seriesNames[hash] = seriesLabels.String()
 				}
 
-				volumes[seriesNames[hash]] += stats.KB << 10 // Return bytes
+				size := stats.KB << 10 // Return bytes
```

Comment:
I don't think we want this here. This will cause the results from `stats` and this function to differ for the same inputs. 

_we do_ want it in the ingester instance.

I think this is handled by the `v3 index reader` already but we should validate and add it as a separate PR.

Reply:
removed